services/models/connectivity/src/main/java/org/eclipse/ditto/services/models/connectivity/OutboundSignal.java

```diff
@@ -60,6 +61,18 @@ default JsonObject toJson(final JsonSchemaVersion schemaVersion, final JsonField
         return toJson(schemaVersion, FieldType.notHidden()).get(fieldSelector);
     }
 
+    /**
+     * Extends the {@link OutboundSignal} by adding the payload mapping used to map the signal.
+     */
+    interface Mappable extends OutboundSignal {
```

Comment:
We should move the Mappable interface to its own file and rename it to MappableOutboundSignal. Nesting it in its super-type confused me at first.